app/assets/javascripts/message.js

```diff
@@ -0,0 +1,50 @@
+$(document).on('turbolinks:load',function(){
+  function buildHTML(message){
+    var content = message.content ? `${message.content}` : "";
+    var img = message.image ? `<img src= ${message.image }>` : "";
+    var html = `<div class="message">
+    <div class="top-message">
+      <div class="top-message__user-name">
+        ${message.user_name}
+      </div>
+      <div class="top-message__date">
+        ${message.date}
+      </div>
+    </div>
+    <div class="bottom-message">
+      <div class="bottom-message__content">
+        ${content}
+      </div>
+      <div class='lower-message__image'>
+        ${img}
+      </div>
+    </div>`
+    return html;
+  }
+  $('#new_message').on('submit', function(e) {
+    e.preventDefault();
+    var message = new FormData(this);
+    var url = $(this).attr('action')
+    $.ajax({
+      url: url,
+      type: "POST",
+      data: message,
+      dataType: 'json',
+      processData: false,
+      contentType: false
+    })
+    .done(function(data){
+      var html = buildHTML(data);
+      $('.messages').append(html)
+      $('#message_content').val('')
```

Comment:
formのresetメソッド使いましょう

Reply:
```suggestion
      $('#message_content')[0].reset();
```
としても機能しないのですが、どうしてでしょうか？
一見.val('')としている方の方が多いのですが、どう違うのでしょうか？

Reply:
自己解決致しました！